templates/docs/document.html

```diff
@@ -19,7 +19,7 @@
     </div>
   </section>
   <div class="l-docs">
-    <aside class="l-docs-sidebar" id="navigation">
+    <aside class="l-docs-sidebar" id="navigation" aria-label="document contents">
```

Comment:
In this case, this is navigation not document contents. I assume the `nav` element is sufficient here?